### PR TITLE
OCPBUGS-1316: Add missing variable reference to rules

### PR DESCRIPTION
**Pull request by Vincent056**
Some of the kubeletconfig rules does not have not contains reference to variable being used in its remediation, this PR adds that reference to those rules. Related BUG: https://issues.redhat.com/browse/OCPBUGS-1316

### CODE REVIEW

File: shared/macros/10-kubernetes.jinja [hunk 1]
```diff
@@ -176,6 +176,10 @@ spec:
         overwrite: true
 {{%- endmacro -%}}
 
+{{%- macro kubenetes_varible_instruction(field_name='', variable_name='') -%}}
```
Comment (rhmdnd) [inline shared/macros/10-kubernetes.jinja:179]: nit: minor misspelling in the macro name (kubernetes_variable_instruction)

**Comment by BhargaviGudi** [comment]
Verification passed with 4.16.0-0.nightly-2024-05-23-173505 + https://github.com/ComplianceAsCode/compliance-operator code + PR https://github.com/ComplianceAsCode/content/pull/12012 code
```
$ oc get rule -n openshift-compliance -o custom-columns=NAME:metadata.name,VARIABLE:metadata.annotations.compliance\\.openshift\\.io/rule-variable --no-headers  | grep upstream | grep eviction
upstream-ocp4-kubelet-eviction-thresholds-set-hard-imagefs-available                         var-kubelet-evictionhard-imagefs-available
upstream-ocp4-kubelet-eviction-thresholds-set-hard-imagefs-inodesfree                        var-kubelet-evictionhard-imagefs-inodesfree
upstream-ocp4-kubelet-eviction-thresholds-set-hard-memory-available                          var-kubelet-evictionhard-memory-available
upstream-ocp4-kubelet-eviction-thresholds-set-hard-nodefs-available                          var-kubelet-evictionhard-nodefs-available
upstream-ocp4-kubelet-eviction-thresholds-set-hard-nodefs-inodesfree                         var-kubelet-evictionhard-nodefs-inodesfree
upstream-ocp4-kubelet-eviction-thresholds-set-soft-imagefs-available                         var-kubelet-evictionsoft-imagefs-available
upstream-ocp4-kubelet-eviction-thresholds-set-soft-imagefs-inodesfree                        var-kubelet-evictionsoft-imagefs-inodesfree
upstream-ocp4-kubelet-eviction-thresholds-set-soft-memory-available                          var-kubelet-evictionsoft-memory-available
upstream-ocp4-kubelet-eviction-thresholds-set-soft-nodefs-available                          var-kubelet-evictionsoft-nodefs-available
upstream-ocp4-kubelet-eviction-thresholds-set-soft-nodefs-inodesfree                         var-kubelet-evictionsoft-nodefs-inodesfree
```

```
CR: upstream-ocp4-cis-node-master-kubelet-eviction-thresholds-set-hard-imagefs-available
  This rule pertains to the imagefs.available setting of the evictionHard section. Remediations for the imagefs.available field will be set to 15% based on variable var_kubelet_evictionhard_imagefs_available.
CCR: upstream-ocp4-cis-node-master-kubelet-eviction-thresholds-set-hard-memory-available
  This rule pertains to the memory.available setting of the evictionHard section. Remediations for the memory.available field will be set to 100Mi based on variable var_kubelet_evictionhard_memory_available.
CCR: upstream-ocp4-cis-node-master-kubelet-eviction-thresholds-set-hard-nodefs-available
  This rule pertains to the nodefs.available setting of the evictionHard section. Remediations for the nodefs.available field will be set to 10% based on variable var_kubelet_evictionhard_nodefs_available.
CCR: upstream-ocp4-cis-node-master-kubelet-eviction-thresholds-set-hard-nodefs-inodesfree
  This rule pertains to the nodefs.inodesFree setting of the evictionHard section. Remediations for the nodefs.inodesFree field will be set to 5% based on variable var_kubelet_evictionhard_nodefs_inodesfree.
CCR: upstream-ocp4-cis-node-worker-kubelet-eviction-thresholds-set-hard-imagefs-available
  This rule pertains to the imagefs.available setting of the evictionHard section. Remediations for the imagefs.available field will be set to 15% based on variable var_kubelet_evictionhard_imagefs_available.
CCR: upstream-ocp4-cis-node-worker-kubelet-eviction-thresholds-set-hard-memory-available
  This rule pertains to the memory.available setting of the evictionHard section. Remediations for the memory.available field will be set to 100Mi based on variable var_kubelet_evictionhard_memory_available.
CCR: upstream-ocp4-cis-node-worker-kubelet-eviction-thresholds-set-hard-nodefs-available
  This rule pertains to the nodefs.available setting of the evictionHard section. Remediations for the nodefs.available field will be set to 10% based on variable var_kubelet_evictionhard_nodefs_available.
CCR: upstream-ocp4-cis-node-worker-kubelet-eviction-thresholds-set-hard-nodefs-inodesfree
  This rule pertains to the nodefs.inodesFree setting of the evictionHard section. Remediations for the nodefs.inodesFree field will be set to 5% based on variable var_kubelet_evictionhard_nodefs_inodesfree.
```

File: shared/macros/10-kubernetes.jinja [hunk 2]
```diff
@@ -176,6 +176,10 @@ spec:
         overwrite: true
 {{%- endmacro -%}}
 
+{{%- macro kubernetes_variable_instruction(field_name='', variable_name='') -%}}
+Remediations for the {{{ field_name }}} field will be set to {{ .{{{ variable_name }}} }} based on variable {{{ variable_name }}}.
```
Comment (yuumasato) [inline shared/macros/10-kubernetes.jinja:180]: Nitpick:
```suggestion
Remediation will set field {{{ field_name }}} to {{ .{{{ variable_name }}} }} based on the variable {{{ variable_name }}}.
```

**Review by yuumasato — APPROVED** [review]

**Pull request merged**